### ISSUE TITLE
[WebToolsE2E][Aspire] There are some errors "Refused to apply style from '[https://localhost:XXX/......']" in the website console pane when running the Aspire project

### ISSUE THREAD

**Issue by Rita003** [dotnet/aspire#6016]
**REGRESSION INFO:** Work in Aspire 8.2.1

**INSTALL STEPS**
1. Clean machine: Win11 x64 23h2 ENU
2. Install SDK 8.0.402
3. Install Aspire 9.0 RC1 [9.0.0-preview.4.24479.1]

**REPRO STEPS**
1. Open CMD, create an Aspire App using following commands:
```
dotnet new aspire -o AspireApp
cd AspireApp/AspireApp.AppHost
```
2. Dotnet run the project in CMD, and open URL in the browser.
`dotnet run`
3. Hit F12 and navigate to Console in the browser development tools.

**Note:**
1. This issue doesn't repro when running the normal project
2. There is a known issue with console pane warnings: [#5279](https://github.com/dotnet/aspire/issues/5279)

**ACTUAL**
There are some errors in the website console pane
![image](https://github.com/user-attachments/assets/4dee700f-5bbe-4336-a105-ffa30d2b6ffd)

**Error Log:**
```
Refused to apply style from '[https://localhost:17067/_content/Microsoft.FluentUI.AspNetCore.Components/Microsoft.FluentUI.AspNetCore.Components.bundle.scp.css'](https://localhost:17067/_content/Microsoft.FluentUI.AspNetCore.Components/Microsoft.FluentUI.AspNetCore.Components.bundle.scp.css%27) because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
localhost/:1  Refused to apply style from '[https://localhost:17067/_content/Microsoft.FluentUI.AspNetCore.Components/Microsoft.FluentUI.AspNetCore.Components.bundle.scp.css'](https://localhost:17067/_content/Microsoft.FluentUI.AspNetCore.Components/Microsoft.FluentUI.AspNetCore.Components.bundle.scp.css%27) because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
```

**Comment by JamesNK** [comment]
I think I've already fixed this.

**Comment by Rita003** [comment]
Verify this issue fixed on the latest Aspire 9.0.0-rc.1.24506.1.
